app/controllers/houses_controller.rb

```diff
@@ -1,11 +1,11 @@
-class HouseController < ApplicationController
+class HousesController < ApplicationController
 
   def index
-    @houses = Houses.all
+    @houses = House.all
   end
 
   def show
-    @houses = Houses.all
+    @house = House.find(:id)
```

Comment:
`:id` alone won't work. You needs to say `params[:id]` so it pulls the id from the URL.